source/docs/guides/EMV_3D_secure.md

```diff
@@ -564,6 +559,338 @@ trnApproved=1
 &3DSecureStatus=Success
 ```
 
+## EMV 3D Secure API
+
+The EMV 3D Secure API provides a set of end points that you can utilize to perform 3DSecure authentication without immediately processing a payment.  This allows the card holder and credit card number to be authenticated and potentially stored or later referenced in a payment transaction.  Our Payments API supports accepting the 3DS Session Data a token to the card data utilized in a previously authenticated 3D Secure transaction.
```

Comment:
Need a comma after 'to be authenticated'

Reply:
supposed to be 'as a token to the card data'?

Reply:
updated